### DISCUSSION TITLE
How do I see an itemized Actions charges?

### DISCUSSION THREAD

**Discussion by andy-brainome** [community/community#27043] [Actions]
<p>My Actions charges blew out my $200 budget two days after it renewed.<br>
How do I see an itemized statement of what is costing me so much?</p>

**Comment by shrink** [comment, marked as answer]
<p>Organisation &gt; <a href="https://github.com/organizations/brainome/settings/billing" rel="noopener nofollow ugc">Billing &amp; plans</a> &gt; Usage this month &gt; “Get usage report”</p>
<p>You’ll then receive an email containing a CSV of individual costs, for example:</p>
<div class="md-table">
<table>
<thead>
<tr>
<th>Date</th>
<th>Product</th>
<th>Repository Slug</th>
<th>Quantity</th>
<th>Unit Type</th>
<th>Price Per Unit</th>
<th>Actions Workflow</th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr>
<td>2021-06-16</td>
<td>shared storage</td>
<td>example/repository</td>
<td>0.0</td>
<td>gb</td>
<td>$0.25</td>
<td></td>
<td></td>
</tr>
<tr>
<td>2021-06-24</td>
<td>actions</td>
<td>example/repository</td>
<td>2</td>
<td>UBUNTU</td>
<td>$0.008</td>
<td>.github/workflows/example.yml</td>
<td></td>
</tr>
</tbody>
</table>
</div><p>There are additional charges for <a href="https://github.com/features/packages#pricing" rel="noopener nofollow ugc">package storage and data transfer out</a> which <em>includes</em> the storage used by Actions (<a href="https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions" rel="noopener nofollow ugc">as described here</a>).</p>

**Comment by andy-brainome** [comment]
<p>Thank you fo the reply <a class="mention" href="/shrink">@shrink</a>. Your response directly answered my question.<br>
As these things usually go - one question leads to another.<br>
I asked it here <a href="https://github.community/t/where-did-all-the-time-host-minutes-go/187207" class="inline-onebox">Where did all the time (host minutes) go?</a><br>
TLDR; <em>The 3000 minutes included with my paid plan are unaccounted for in the Billing &amp; plans “Get usage report”</em>  Whomever design the report only considered billable time rather than “included” time.</p>

**Comment by shrink** [comment]
<p><a class="mention" href="/andy-brainome">@andy-brainome</a> I have reviewed my own usage report and I can see that included minutes for <em>private</em> repositories are captured[1]. As I understand it, the report captures the usage and cost <em>before</em> considering any included minutes – however, it doesn’t capture Action usage for public repositories.</p>
<p>The minutes used by a specific Run of a Workflow are listed on the Workflow Run page itself. Navigate to the repository &gt; Actions &gt; Pick a Workflow Run &gt; Click on “Billable time” in the Workflow Stats. A common pitfall when measuring Workflow minutes usage is the difference between a Workflow and a job. A matrix strategy, for example, can trigger hundreds of jobs: it’s possible for a 1-hour Workflow to use thousands of minutes.</p>
<p>If you’re confident that the minutes in your usage report aren’t representative of the minutes used, I definitely recommend reaching out to <a href="https://github.com/support" rel="noopener nofollow ugc">GitHub Support</a>. They’re very responsive and helpful, you should be able to get this squared away with their direct insight into your account :slight_smile:</p>
<p>[1] I am well within my limit and I see the following:</p>
<pre><code class="lang-auto">2021-06-28,actions,shrink/tests-actions,33,UBUNTU,$0.008,.github/workflows/skip-job.yml,
2021-06-28,actions,shrink/tests-actions,51,UBUNTU,$0.008,.github/workflows/default-input.yml,
</code></pre>
